README.md

```diff
@@ -123,25 +123,52 @@ These tasks are also performed during the [travis](.travis.yml) runs.
 All emails sent by the application can be inspected via [mailhog](https://github.com/mailhog/MailHog)
 at [http://localhost:8025/](http://localhost:8025/)
 
-## Accessing the database
+## Database
+
+### Database migrations
+
+Out of the box, the database should be in a usable state for local development. For production changes to the database,
+doctrine migration scripts may be used. Migration scripts are stored in the *migrations* directory of
+the [FundraisingStore repository](https://github.com/wmde/FundraisingStore/tree/master/migrations).
+
+The scripts from the FundraisingStore repository can be executed through FundraisingFrontend by running these commands:
+
+To execute a specific script, run the following command and add the version number of the migration script you want to use.
+As an example, executing `migrations/Version20180612000000.php` would look like this:
+
+```
+make migration-execute MIGRATION_VERSION=20180612000000
+```
+
+You can also revert a script (if implemented) through an equivalent `migration-revert` command:
+
+```
+make migration-revert MIGRATION_VERSION=20180612000000
+```
+
+Note that Doctrine creates its own `doctrine_migration_versions` table where it stores the status of individual migrations.
+If you run into issues and want to reset the state of a migrations, it's best to check that table directly or use the `versions`
+command from doctrine-migrations which supports `--add` and `--delete` paramters:
+
+```
+vendor/wmde/fundraising-store/vendor/doctrine/migrations/bin/doctrine-migrations migrations:version
+```
 
 ### Accessing the database from a Docker image
 
-The database container of the Docker development environment is not exposed to the outside. If you want to connect to 
-the default fundraising frontend database using the MySQL command line client, you need first to find out the Docker 
-network name where the database is running in. With the command
+If you want to access the database via the terminal, the easiest way is to first start the docker containers via `docker-compose up`,
+you can then connect to the database via:
 
-    docker network ls
+    docker exec -it fundraisingfrontend_database_1 /bin/bash
 
-you can list all networks. There will be one network name ending in `fundraising_proxy` (the prefix is probably the 
-directory name where you checked out this repository). Copy the full network name and use it instead of the 
-placeholder `__NETWORK_NAME__` in the following command to run the MySQL command line client:  
+This will open a bash terminal in the database container, from there you can connect to MySQL via:
 
-    docker run -it --link spenden2_database_1:mysql --net __NETWORK_NAME__ --rm mysql:5.6 \
```

Comment:
As you can see here, my folder is called 'spenden2' instead of 'FundraisingFrontend'. So we probably need additional instructions on how to determine the container name.